### PR TITLE
fixed a few typos and forgotten pipi decays

### PR DESCRIPTION
**Pull request by vbellee**

### CODE REVIEW

**Comment by vlisovsk** [comment]
I am afraid not all the mentions of pions are removed. At least one survived in the very beginning:
> In addition the kaon and the pion should originate from exactly the same point in space.
(Well, that could be referred to a bachelor pion from D*, but then the explanation needs to be modified)

File: first-analysis-steps/decay-tree-fitter.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ To this branch we can now apply the `TupleToolDecayTreeFitter`.
 ```python
 dtt.Dstar.addTupleTool('TupleToolDecayTreeFitter/ConsD')
```
Comment (vlisovsk) [inline first-analysis-steps/decay-tree-fitter.md]: Should we explain here that `ConsD` is a random name we decided to assign?

Comment (vbellee) [reply first-analysis-steps/decay-tree-fitter.md]: Yes, we should. Fixing this now

**Comment by vbellee** [comment]
I'm fixing the forgotten mention.

File: first-analysis-steps/code/decay-tree-fitter/ntuple_DTF1.py [hunk 2]
```diff
@@ -24,15 +24,15 @@
...
 
 # add another fitter, this time we will change a mass hypothesis
-dtt.Dstar.addTupleTool('TupleToolDecayTreeFitter/ConsDpipi')
-dtt.Dstar.ConsDpipi.constrainToOriginVertex = True
-dtt.Dstar.ConsDpipi.Verbose = True
-dtt.Dstar.ConsDpipi.daughtersToConstrain = ['D0']
+dtt.Dstar.addTupleTool('TupleToolDecayTreeFitter/ConsDKpi')
+dtt.Dstar.ConsDKpi.constrainToOriginVertex = True
+dtt.Dstar.ConsDKpi.Verbose = True
+dtt.Dstar.ConsDKpi.daughtersToConstrain = ['D0']
```
Comment (chrisburr) [inline first-analysis-steps/code/decay-tree-fitter/ntuple_DTF1.py]: I think this should be `['D0', 'D~0']` (and also for `dtt.Dstar.ConsD.daughtersToConstrain` a few lines above)

Comment (chrisburr) [reply first-analysis-steps/code/decay-tree-fitter/ntuple_DTF1.py]: For the record it's not needed, [the absolute PDG ID is constrained](https://gitlab.cern.ch/lhcb/Analysis/blob/master/Phys/DecayTreeTuple/src/TupleToolDecayTreeFitter.cpp#L671).

**Pull request merged**